entc/integration/type_test.go

```diff
@@ -117,6 +117,7 @@ func Types(t *testing.T, client *ent.Client) {
 	require.NoError(err)
 	require.False(exists)
 
+	// err on MaxLen validator
```

Comment:
Move this comment to the `require.Error` call:
```suggestion
	require.Error(err. "MaxLen validator should reject this operation")
```